static-site/src/components/ListResources/index.tsx

```diff
@@ -147,12 +160,12 @@ export default ListResourcesResponsive
 
 
 function SortOptions({sortMethod, changeSortMethod}) {
-  function onChangeValue(event) {
-    changeSortMethod(event.target.value);
+  function onChangeValue(event:FormEvent<HTMLInputElement>): void {
+    changeSortMethod(event.currentTarget.value);
```

Comment:
The change from `target` to `currentTarget` doesn't have any functional impact?

Reply:
Not that I've noticed. The Typescript compiler tells me `event.target` is not defined, too.